interface/super/edit_globals.php

```diff
@@ -645,22 +645,6 @@ function checkBackgroundServices()
                                                     echo "</option>\n";
                                                 }
                                                 echo "  </select>\n";
-                                            } elseif ($fldtype == GlobalSetting::DATA_TYPE_MULTI_DASHBOARD_CARDS) {
-                                                $res = sqlStatement("SELECT * FROM dashboard_card_visibility");
-                                                echo "  <select multiple class='form-control' name='form_{$i}[]' id='form_{$i}[]' size='3'>\n";
-                                                while ($row = sqlFetchArray($res)) {
-                                                    echo "   <option value='" . attr($row['card_abrev']) . "'";
-                                                    foreach ($glarr as $glrow) {
-                                                        if ($glrow['gl_value'] == $row['card_abrev']) {
-                                                            echo " selected";
-                                                            break;
-                                                        }
-                                                    }
-                                                    echo ">";
-                                                    echo xlt($row['card_name']);
-                                                    echo "</option>\n";
-                                                }
-                                                echo "  </select>\n";
```

Comment:
@juggernautsei I don't understand what is going here. This is the preferred way you strove for! Just needed to hard code the options that current are in the new table then  revert table.
Did you accidently push an older version?

Reply:
Accident.

Reply:
I got the code and will start a new PR for clarity sake.